lab-yunjoo/lib/task-crud.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const debug = require('debug')('rito:note-crud');
+const mongoose = require('mongoose');
+mongoose.Promise = require('bluebird');
+const Task = require('../model/task');
+const noteCrud = require('./note-crud');
+const AppError = require('./app-error');
+
+exports.createTask = function(reqBody){
+  debug('creating task');
+  return new Promise((resolve, reject) => {
+    if(! reqBody.noteId)
```

Comment:
I probably wouldn't put a space between the '!' and the condition--it's obviously a code style thing, but I've never seen anyone else do it before and it looks confusing.